docs/index.rst

```diff
@@ -42,6 +43,18 @@ Information on recent releases and citation information for
 Sherpa is available using the Digital Object Identifier (DOI)
 `10.5281/zenodo.593753 <https://doi.org/10.5281/zenodo.593753>`_.
 
+Acknowledging or Citing Sherpa
+==============================
+
+If you use Sherpa for work/research presented in a publication please cite the Sherpa papers:
+
+`Sherpa Paper 2024 <https://arxiv.org/abs/2409.10400>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2024arXiv240910400S/exportcitation>`_)
+
+`Sherpa Paper 2007 <https://articles.adsabs.harvard.edu/full/2007ASPC..376..543D>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2007ASPC..376..543D/exportcitation>`_)
+
+`Sherpa Paper 2001 <https://arxiv.org/abs/astro-ph/0108426>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2001SPIE.4477...76F/exportcitation>`_)
```

Comment:
The readthedocs build fails with 
```
/home/docs/checkouts/readthedocs.org/user_builds/sherpa/checkouts/2153/docs/index.rst:47: WARNING: Duplicate explicit target name: "ads bibtex".
/home/docs/checkouts/readthedocs.org/user_builds/sherpa/checkouts/2153/docs/index.rst:47: WARNING: Duplicate explicit target name: "ads bibtex".
```

I think (but did not test) that making the anonymous links (without an explicit entry in sphinx directory of targets shuld work):
```suggestion
`Sherpa Paper 2024 <https://arxiv.org/abs/2409.10400>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2024arXiv240910400S/exportcitation>`__)

`Sherpa Paper 2007 <https://articles.adsabs.harvard.edu/full/2007ASPC..376..543D>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2007ASPC..376..543D/exportcitation>`__)

`Sherpa Paper 2001 <https://arxiv.org/abs/astro-ph/0108426>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2001SPIE.4477...76F/exportcitation>`__)
```